drivers/virtualbox/virtualbox.go

```diff
@@ -452,19 +452,19 @@ func (d *Driver) GetIP() (string, error) {
 	if s != state.Running {
 		return "", drivers.ErrHostIsNotRunning
 	}
-	cmd, err := drivers.GetSSHCommandFromDriver(d, "ip addr show dev eth1")
+	output, err := drivers.RunSSHCommandFromDriver(d, "ip addr show dev eth1")
 	if err != nil {
 		return "", err
 	}
 
-	// reset to nil as if using from Host Stdout is already set when using DEBUG
-	cmd.Stdout = nil
+	var buf bytes.Buffer
+	buf.ReadFrom(output.Stdout)
 
-	b, err := cmd.Output()
-	if err != nil {
+	if len(buf.Bytes()) == 0 {
 		return "", err
```

Comment:
What did you assign to `err` before? :) In case of `== 0` it sounds like you're returning an empty err.